extension/goplugin/schemas_test.go

```diff
@@ -623,9 +623,9 @@ var _ = Describe("Schemas", func() {
 		)
 
 		BeforeEach(func() {
-			if os.Getenv("MYSQL_TEST") != "true" {
+			//if os.Getenv("MYSQL_TEST") != "true" {
 				Skip("Locks are only valid in MySQL")
-			}
+			//}
```

Comment:
I made a comment offline that you do not need to fix failed tests in this context because these tests are not maintained for a long time. However, this way of commenting out is not good. You should delete those two lines, then make a comment something like `TODO: tests should be fixed`